controllers/fenceagentsremediation_controller.go

```diff
@@ -129,6 +132,13 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		return emptyResult, nil
 	}
 
+	// Check NHC timeout annotation
+	if isTimedOutByNHC(far) {
+		r.Log.Info("FAR remediation was stopped by Node Healthcheck Operator")
+		// TODO: update status and return its error
+		return emptyResult, errors.New(errorNhcTimedOut)
```

Comment:
why returning an error?

Reply:
> the CR will be stopped here until the NHC annotation would be removed from the CR

exactly, and removing the annotation will trigger a reconcile automatically, no need to requeue ourself 🙂